thirdparty/vulkan/registry/update_deps.py

```diff
@@ -0,0 +1,679 @@
+#!/usr/bin/env python
```

Comment:
Thirdparty python or shell scripts are not needed in our copy, we should only keep the files that we use. I'll have a look to refine further.

Reply:
Fixed by #31102.